src/components/Claim.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import tw, { styled } from "twin.macro";
+
+import Button from "./Button";
+import UnstyledHeading from "./Heading";
+import { ButtonWrapper } from "./Wrappers";
+import { ReactComponent as UnstyledLogo } from "../assets/kpi-frame.svg";
+import { useClaim } from "../hooks";
+
+type ClaimProps = {
+  onCancel?: () => void;
+};
+
+const Claim: React.FC<ClaimProps> = ({ onCancel }) => {
+  const { claim, claimed } = useClaim();
+
+  return (
+    <Wrapper>
+      <Header>
+        <Logo dimmed={!claimed} />
+        <Heading level={1}>
+          {claimed
+            ? "Your options have been claimed."
+            : "Ready to claim your options?"}
+        </Heading>
+      </Header>
+      <Content>
+        <OptionName>uTVL-Jun 30</OptionName>
+        <Metrics>
+          <div>
+            <Label>Quantity</Label>
+            <Value>100</Value>
+          </div>
+          <div>
+            <Label>Expiry</Label>
+            <Value>June 30, 2021</Value>
+          </div>
+          <div>
+            <Label>Current Payout</Label>
+            <Value>$1000</Value>
+          </div>
+          <div>
+            <Label>Min. Payout</Label>
+            <Value>$2368</Value>
```

Comment:
I guess these values become dynamic later? Will they show dollar values or UMA values?

Reply:
Those will be dynamic! They will show dollar value based on how many UMA the options can be redeemed for